core/src/main/java/org/infinispan/configuration/cache/EncodingConfigurationBuilder.java

```diff
@@ -65,6 +82,11 @@ public EncodingConfiguration create() {
 
    @Override
    public void validate(GlobalConfiguration globalConfig) {
+      if (mediaType != null) {
+         CONFIG.ignoringSpecificMediaTypes();
```

Comment:
Why? This mediatype is just a cascade switch to set both key and value at the same time (less typing), and it takes precedence over anything that is set individually.

>  And also shouldn't we only log this if the key or value media types were explicitly defined?

Yes, I will change it, otherwise the yellow lines will keep popping up in the console...